api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerAuthenticationTls.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+
+/**
+ * Configures the Kafka Mirror Maker authentication
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class KafkaMirrorMakerAuthenticationTls extends KafkaMirrorMakerAuthentication {
+    private static final long serialVersionUID = 1L;
+
+    public static final String TYPE_TLS = "tls";
+
+    private CertAndKeySecretSource certificateAndKey;
+
+    @Description("Must be `" + TYPE_TLS + "`")
+    @Override
+    public String getType() {
+        return TYPE_TLS;
+    }
+
+    @Description("Certificate and private key pair for TLS authentication.")
+    public CertAndKeySecretSource getCertificateAndKey() {
```

Comment:
Again, it's a reference to a `Secret`, it's not the cert and private key directly.

Reply:
I recommend "Reference to the `secret` which holds the certificate and private key pair."